### PR TITLE
Default timezone

### PR DESCRIPTION
**Pull request by hassanrafi39**
A important feature for customized websites for any region.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/php/functions/time/index.md [hunk 1]
```diff
@@ -16,3 +16,21 @@ echo time();
 ```
 #### More Information:
 * <a href="https://secure.php.net/manual/en/function.time.php">php.net time() manual</a>
+
+## Default timezone
+
+You can set or get the settings using `date_default_timezone_set()` and `date_default_timezone_get()`.
```
Comment (co16353sidak) [inline guide/english/php/functions/time/index.md]: As a suggestion make it more clear to the reader, what settings you are peering here.
you can make changes as follows:-

`date_default_timezone_set()` :- Used to set the default time zone

Similarly for `date_default_timezone_get()`

File: guide/english/php/functions/time/index.md [hunk 2]
```diff
@@ -16,3 +16,21 @@ echo time();
...
+
+### Example
+```php
+<?php
+date_default_timezone_set(‘Asia/Dhaka’);  //To set the time zone.
+$variable = date_default_timezone_get();  //To get the timezone.
+```
```
Comment (co16353sidak) [inline guide/english/php/functions/time/index.md]: Here you haven't echoed the `$variable` so technically there won't be any output, so it is suggested to add an echo keyword such as
`echo $variable`

**Review by theNaut1lus — CHANGES_REQUESTED** [review]
@hassanrafi39 I have requested a few changes.

**Comment by RandellDawson** [comment]
Hi @hassanrafi39 

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened or create a new PR, if you still wish to make the changes you originally added plus any additional change requests asked of you.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding!

**Pull request closed without merging**